ac_runtime-api/modules/txpayment.adoc

```diff
@@ -73,3 +75,67 @@ where::::
 *** latexmath:[f_l] is the length fee, the amount paid for the encoded length (in bytes) of the extrinsic.
```

Comment:
> where::::
> *** ...

Is the formatting correct here?

Reply:
This was from a previous commit. It is correct, it lets you nest deeper and provides the right indentation. removing it flattens the indentation